diffkemp/simpll/Result.h

```diff
@@ -79,6 +79,20 @@ struct FunctionInfo {
     }
 };
 
+/// Type for storing information about definition of an analysed
+// 'object' (eg. macro).
+struct Definition {
```

Comment:
This class is very similar to `CallInfo` (which also contains a name and a location) so I'm wondering if it makes sense to eventually merge them (something like `CodeLocation`). I'm not saying that we should do it now but it's something to think about.

Reply:
There is also [`FunctionInfo`](https://github.com/diffkemp/diffkemp/blob/e6cded9aefd2e51ada26c972f499fe08d442d789/diffkemp/simpll/Result.h#L58) which also contains name and location and other things. So I am thinking that I could rename in this PR the `Definition` struct to `CodeLocation`  and make another PR where I could refactor the `CallInfo` and `FunctionInfo` structs so they extends the `CodeLocation`, if that makes sense to you?

Reply:
Yes, that makes sense. `CodeLocation` sounds like something that could be reused in multiple places (definitions, call sites, etc.).

Reply:
It is renamed.

Reply:
> ... make another PR where I could refactor the `CallInfo` and `FunctionInfo` structs so they extends the `CodeLocation` ...

Implemented in #348